driver/driver.go

```diff
@@ -349,11 +349,11 @@ func (d *ContrailDriver) CreateEndpoint(req *network.CreateEndpointRequest) (
 	}
 
 	contrailIP, err := d.controller.GetOrCreateInstanceIp(contrailNetwork, contrailVif)
-	instanceIP := contrailIP.GetInstanceIpAddress()
-	log.Infoln("Retreived instance IP:", instanceIP)
 	if err != nil {
 		return nil, err
 	}
+	instanceIP := contrailIP.GetInstanceIpAddress()
+	log.Infoln("Retreived instance IP:", instanceIP)
```

Comment:
Retreived -> Retrieved

Reply:
I now know why I made this typo in the first place: it's because I confused it with spelling of `receive`.